.github/PULL_REQUEST_TEMPLATE/bugfix.md

```diff
@@ -0,0 +1,40 @@
+---
+name: 🐛 Bug Fix
+about: Did you fix something that did not work as expected?
+---
+
+<!---
+Thanks for filing a pull request 😄 ! Before you submit, please read the following:
+
+Search open/closed issues before submitting since someone might have pushed the same thing before!
```

Comment:
This should probably be search open/closed PRs

Reply:
I missed this by purpose but in the end it may have been better to say "Search open/closed _similar issues and pull requests_ before submitting[...]", but forgot to apply this commit so it ended up like it is now.

Reply:
I would need a review from you for this as I'm making the last adjustments for my next PR